Code/src/main/java/nl/utwente/group10/haskell/expr/Ident.java

```diff
@@ -0,0 +1,42 @@
+package nl.utwente.group10.haskell.expr;
+
+import nl.utwente.group10.haskell.env.Env;
+import nl.utwente.group10.haskell.exceptions.HaskellTypeError;
+import nl.utwente.group10.haskell.hindley.GenSet;
+import nl.utwente.group10.haskell.type.Type;
+
+/**
+ * Expression contained in the environment.
```

Comment:
Isn't it more of a *reference* to an expression (function) contained in the environment?